source/compatibility.tex

```diff
@@ -1881,6 +1881,19 @@
 Valid \CppIII{} code that attempts to explicitly instantiate a container using
 a user-defined type with no default constructor may fail to compile.
 
+\diffref{container.requirements.general}
+\change
+Containers now access their allocators through the \tcode{allocator_traits}
+template.
+\rationale
+Simplifies writing new allocators.
+\effect
+\tcode{allocator_traits} supplies default definitions for many allocator type
+names and operations. Valid \CppIII{} code that follows the original
+allocator-aware container requirements may not support allocators written to
+the simpler set of requirements in this standard, and may propagate allocators
```

Comment:
IMO allocator propagation deserves a separated paragraph since corresponding change in C++11 doen't seemly simplify user codes.
Library supports for stateful allocators were essentially conditionally-supported extensions in C++03, while being precisely required and specified since C++11. Incompatibility may come from the conflict between standard and (pre-standard) extented mechanisms (only when using stateful allocators with standard containers).